programs/bench.c

```diff
@@ -407,8 +407,6 @@ static void BMK_benchCLevel(void* srcBuffer, size_t benchedSize,
     if (g_displayLevel == 1 && !g_additionalParam)
         DISPLAY("bench %s %s: input %u bytes, %u seconds, %u KB blocks\n", LZ4_VERSION_STRING, LZ4_GIT_COMMIT_STRING, (U32)benchedSize, g_nbSeconds, (U32)(g_blockSize>>10));
 
```

Comment:
Why is this bound remove ?
Is there some kind of guarantee that `cLevelLast >= level` ? 
In which case, it should be stated with an `assert()`.

Or does it not matter ?
In which case, what happen if a user defines an end level lower than start level ?
such as `zstd -b3e1` for example ?